api/services_cluster.go

```diff
@@ -0,0 +1,100 @@
+package api
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+
+	"github.com/canonical/lxd/lxd/response"
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/lxd/shared/logger"
+	"github.com/canonical/microcluster/rest"
+	"github.com/canonical/microcluster/state"
+	"github.com/gorilla/mux"
+
+	"github.com/canonical/microcloud/microcloud/api/types"
+	"github.com/canonical/microcloud/microcloud/service"
+)
+
+// ServicesClusterCmd represents the /1.0/services/cluster/{name} API on MicroCloud.
+var ServicesClusterCmd = func(sh *service.Handler) rest.Endpoint {
+	return rest.Endpoint{
+		AllowedBeforeInit: true,
+		Name:              "services/cluster/{name}",
+		Path:              "services/cluster/{name}",
+
+		Delete: rest.EndpointAction{Handler: authHandler(sh, removeClusterMember), AllowUntrusted: true},
```

Comment:
A general question about `AllowUntrusted: true`. Does the security in MicroCloud come from the fact that only a root user can connect to the Unix socket to trigger this action?

Reply:
That's the case for now, I believe we can add an auth handler to these with the secure key exchange mechanism.

Reply:
Ok thanks!